### PR TITLE
Add help file for rhel image

### PR DESCRIPTION
**Pull request by punkisdead**
Provide help file for Docker images to maintain Atomic container best practices.

https://github.com/projectatomic/container-best-practices/blob/master/creating/help.adoc

### CODE REVIEW

File: Dockerfile.rhel7 [hunk 1]
```diff
@@ -45,6 +48,10 @@ RUN set -x && \
     yum-config-manager --enable rhel-7-server-thirdparty-oracle-java-rpms && \
     yum -y update-minimal --security --sec-severity=Important --sec-severity=Critical --setopt=tsflags=nodocs && \
     yum -y install --setopt=tsflags=nodocs tar java-1.8.0-oracle-devel && \
+    yum-config-manager --enable rhel-7-server-optional-rpms && \
+    ### help markdown to man conversion
+    yum -y install golang-github-cpuguy83-go-md2man && go-md2man -in help.md -out help.1 && \
+    yum -y remove golang-github-cpuguy83-go-md2man && rm -f help.md && \
```
Comment (whyjustin) [inline Dockerfile.rhel7:53]: I think ```go-md2man``` can be removed too

File: Dockerfile [hunk 2]
```diff
@@ -18,7 +18,7 @@ MAINTAINER Sonatype <cloud-ops@sonatype.com>
 # Atomic Labels
 LABEL name="Nexus Repository Manager" \
       vendor="Sonatype" \
-      version="3.0.2-02" \
+      release="3.0.2-02" \
       url="https://sonatype.com" \
       summary="The Nexus Repository Manager server \
           with universal support for popular component formats." \
```
Comment (whyjustin) [inline Dockerfile:24]: Are the ```golang-github-cpuguy83-go-md2man``` packages available for CentOS? We should do help on both versions.

File: Dockerfile [hunk 3]
```diff
@@ -18,7 +18,7 @@ MAINTAINER Sonatype <cloud-ops@sonatype.com>
 # Atomic Labels
 LABEL name="Nexus Repository Manager" \
       vendor="Sonatype" \
-      version="3.0.2-02" \
+      release="3.0.2-02" \
```
Comment (whyjustin) [inline Dockerfile]: Missing version. Replacing should fix CI.

File: help.md [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+`stop`
+
+Stops the installed container
+
+`uninstall`
+
+Removes the installed container, not the image
+
+The container itself consists of:
+    - RHEL7 base image
```
Comment (whyjustin) [inline help.md]: If we can provide help for the CentOS version then this should indicate it could be either.

File: help.md [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+
+`STOP=`
+
+  LABEL STOP='docker stop ${NAME}'
+
+`Name=`
+
+The registry location and name of the image. For example, Name="Nexus Repository Manager".
+
+`Version=`
```
Comment (whyjustin) [inline help.md:60]: I don't like this practice of providing the RHEL version as version. For now, let's only include the version tag which should be ```The Nexus Repository Manager version from which the container was built. For example, Version="3.0.2-02".``` and remove the Release tag from the help file and from the Dockerfiles.

**Review by whyjustin — CHANGES_REQUESTED** [review]

File: Dockerfile.rhel7 (hunk 1 above)
Comment (punkisdead) [reply Dockerfile.rhel7:53]: `go-md2man` is provided by `golang-github-cpuguy83-go-md2man` it's not a separate package.

File: Dockerfile (hunk 2 above)
Comment (punkisdead) [reply Dockerfile:24]: It appears that `golang-github-cpuguy83-go-md2man` is available in the extras repository. `go-md2man` is not a separate package, but is provided by the `golang-github-cpuguy83-go-md2man` package.

File: help.md (hunk 5 above)
Comment (punkisdead) [reply help.md:60]: :+1:

File: Dockerfile.rhel7 (hunk 1 above)
Comment (whyjustin) [reply Dockerfile.rhel7:53]: 👍

File: help.md [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+`stop`
+
+Stops the installed container
+
+`uninstall`
+
+Removes the installed container, not the image
+
+The container itself consists of:
+    - Centos 7 base image
```
Comment (whyjustin) [inline help.md]: I would keep one help file and say a ```Linux base image```. This prevents the need for maintaining multiple helps.

File: help.md [hunk 7]
```diff
@@ -0,0 +1,70 @@
...
+`run`
+
+Starts the installed container with selected privileges to the host.
+
+`stop`
+
+Stops the installed container
+
+The container itself consists of:
+    - Linux base image
```
Comment (whyjustin) [inline help.md:23]: - Oracle Java JDK
- Nexus Repository Manager

**Review by whyjustin — CHANGES_REQUESTED** [review]

**Review by whyjustin — APPROVED** [review]
:shipit:

**Pull request merged**